public/keymaps/m/marksard_rhymestone_rev1.json

```
@@ -0,0 +1,35 @@
+{
+    "version": 1,
+    "notes": "",
+    "documentation": "\"This file is a QMK Configurator export. You can import this at <https://config.qmk.fm>. It can also be used directly with QMK's source code.\n\nTo setup your QMK environment check out the tutorial: <https://docs.qmk.fm/#/newbs>\n\nYou can convert this file to a keymap.c using this command: `qmk json2c {keymap}`\n\nYou can compile this keymap using this command: `qmk compile {keymap}`\"\n",
+    "keyboard": "marksard/rhymestone/rev1",
+    "keymap": "default",
+    "layout": "LAYOUT_ortho_4x10",
```

This will help me know if this file needs to be updated if you change the keymap on the qmk_firmware repository.

```suggestion
    "commit": "6e32dd123ff0ff49768f43c97bbc10836ff8a629",
    "layout": "LAYOUT_ortho_4x10",
```